src/Domain/Creators/SphereTimeDependentMaps.cpp

```diff
@@ -115,17 +136,28 @@ void TimeDependentMapOptions::build_maps(const std::array<double, 3>& center,
   shape_map_ = ShapeMap{center,         initial_l_max_,
                         initial_l_max_, std::move(transition_func),
                         shape_name,     size_name};
+
+  rigid_translation_map_ = TranslationMap{translation_name};
+
+  if (translation_transition_radii.has_value()) {
```

Comment:
And then here you just always have a rigid translation map and a falloff translation map.